lib/gulp-tasks/file-globbing.js

```diff
@@ -0,0 +1,130 @@
+/**
```

Comment:
I would say this file does more things than just globbing. It takes files and injects them into html file. Globbing means just to select files by specific pattern using stars like `app/*.js`,`**/*.js`.